### PR TITLE
Delete plant variety

### LINKED ISSUE #28: Delete a Specific Plant Variety
As an Admin
In order to protect the integrity of the site
I want to delete a plant variety

Acceptance Criteria:
- The Admin will be able to select a plant to delete from the Plants index page
- The Admin will be prompted wiht a confirmation of deleteing the plant variety
- When the plant is deleted, it should display a confirmation message

Usage:
- From the Plant page for the plant in question:
- Admin will click the "Delete" button
- Admin will be prompted to confirm the deletion
- Redirected to index page with confirmation message of deleted plant

### PR DESCRIPTION
**Pull request by kelleystephens**
Functionality has been added to delete a plant variety from a list of multiple varieties.

resolves #28

### CODE REVIEW

File: config/database.yml.example [hunk 1]
```diff
@@ -1,85 +0,0 @@
-# PostgreSQL. Versions 8.2 and up are supported.
```
Comment (elizamarcum) [inline config/database.yml.example:1]: This file shouldn't be deleted.

File: app/controllers/varieties_controller.rb [hunk 2]
```diff
@@ -16,6 +16,11 @@ def create
     end
   end
 
+  def destroy
+    Variety.find_by(id: params[:id]).destroy
```
Comment (elizamarcum) [inline app/controllers/varieties_controller.rb]: You can just use `Variety.find(params[:id]).destroy` here.

**Comment by elizamarcum** [comment]
@kelleystephens and @Courey: Just a few changes needed

**Comment by kelleystephens** [comment]
@elizabrock & @Courey  destroy method has been updated and added the database.yml.example file back into the config folder.

**Pull request merged**